assets/src/edit-story/app/media/media3p/api/test/useMedia3pApi.js

```diff
@@ -40,7 +40,7 @@ describe('useMedia3pApi', () => {
       media: [
         {
           name: 'media/unsplash:1234',
-          provider: 'UNSPLASH',
+          provider: 'unsplash',
```

Comment:
I think the response uses the uppercase UNSPLASH?

Reply:
Done.